### PR TITLE
[Cloud Tasks] Move samples to new folder

### PR DESCRIPTION
**Pull request by averikitsch**
This is to keep samples consistent with other languages.

### CODE REVIEW

File: tasks/create_http_task_test.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
 TEST_QUEUE_NAME = os.getenv('TEST_QUEUE_NAME', 'my-appengine-queue')
 
 
-def test_create_task():
-    url = 'https://' + TEST_PROJECT_ID + '.appspot.com/example_task_handler'
```
Comment (andrewsg) [inline tasks/create_http_task_test.py]: Why this change in particular?

Comment (averikitsch) [reply tasks/create_http_task_test.py]: It can be any endpoint, so I didn't want someone to think that there was an actual app engine handler.
